tasks/nightwatch.js

```diff
@@ -12,13 +12,31 @@ function toStdErr(data) {
 	process.stderr.write(data.toString());
 }
 
+function filterBrowsers (env, broken) {
+	if (!broken) {
```

Comment:
can we log a warning with the browsers we're filtering out? So devs know their test didn't actually run on all the browsers they expected it to